src/main/java/mate/academy/config/SecurityConfig.java

```diff
@@ -22,4 +26,27 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
         auth.userDetailsService(userDetailsService)
                 .passwordEncoder(passwordEncoder);
     }
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http
+                .sessionManagement()
+                .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+                .and()
+                .authorizeRequests()
+                .antMatchers("/register", "/login", "/inject")
+                .permitAll()
+                .antMatchers(HttpMethod.DELETE)
+                .hasRole(Role.RoleName.ADMIN.name())
+                .anyRequest()
+                .authenticated()
+                .and()
+                .formLogin()
+                .permitAll()
+                .and()
+                .httpBasic()
+                .and()
+                .csrf()
+                .disable();
```

Comment:
you are not applying jwt token anywhere here?

Reply:
missed this , ths